sql/2021/performance/LCP_element.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+# LCP element node
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT(payload, "$._performance.lcp_elem_stats[0].nodeName") AS lcp_node,
```

Comment:
The `JSON_EXTRACT` function will return values with quotes, eg `"IMG"`, rather than the string value itself, eg `IMG`. To omit the quotes, use the `JSON_EXTRACT_SCALAR` function.
```suggestion
  JSON_EXTRACT_SCALAR(payload, "$._performance.lcp_elem_stats[0].nodeName") AS lcp_node,
```

Reply:
Is that the only thing it does, @rviscomi? Because the csv extract didn't result in quotes so I feel like the point is moot?